### PR TITLE
feat: add tracker

### PR DESCRIPTION
**Pull request by dogodo-cc**
Re: #

### Changelog

* add tracker for particle and .fbx

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: editor/inspector/assets/fbx/index.js [hunk 1]
```diff
@@ -149,3 +149,28 @@ exports.methods = {
         }
     },
 };
+
+
+exports.listeners = {
+    async 'track'(event) {
```
Comment (arsen2010) [inline editor/inspector/assets/fbx/index.js]: 不需要 async ; 接口名称叫 metric 比较好，更 `Editor.Metrics.xxx` 接近

File: editor/inspector/components/particle-system.js [hunk 2]
```diff
@@ -699,3 +699,59 @@ exports.style = /* css */`
...
+            'sizeOvertimeModule.enable': 'A100004',
+            'rotationOvertimeModule.enable': 'A100005',
+            colorOverLifetimeModule: 'A100006',
+            textureAnimationModule: 'A100007',
+            'limitVelocityOvertimeModule.enable':'A100008',
+            'trailModule.enable': 'A100009',
+            'renderer.useGPU': 'A1000010',
+        };
+
+        if (Object.keys(trackMap).some(key => dump.path.endsWith(key))) {
```
Comment (arsen2010) [inline editor/inspector/components/particle-system.js]: 这个判断可以不需要

File: editor/inspector/assets/fbx/index.js (hunk 1 above)
Comment (dogodo-cc) [reply editor/inspector/assets/fbx/index.js]: async 已经删除，其实这样的问题应该是放到 eslint的规范去讨论，如果大家规定好不能有多余的 async 那么就直接加入规则，一般在 code review 不进行代码规范的修改，提升效率。

命名我保留意见，因为 track 基本是埋点的首选动词，Editor.Metrics._trackEventWithTimer 从接口这边也体现了 track

File: editor/inspector/components/particle-system.js (hunk 2 above)
Comment (dogodo-cc) [reply editor/inspector/components/particle-system.js]: 好，已经修改，只需要一次遍历

**Review by arsen2010 — APPROVED** [review]

File: editor/inspector/assets/fbx/index.js (hunk 1 above)
Comment (arsen2010) [reply editor/inspector/assets/fbx/index.js]: 在没有 eslint 的时候，靠人肉检查，看到一处就提一处

**Review by VisualSJ — APPROVED** [review]

**Pull request merged**